src/applications/claims-status/tests/e2e/fixtures/mocks/mock-api/claims/claims-list.json

```diff
@@ -88,6 +88,72 @@
         "updatedAt": "2023-03-30T18:19:29.857Z"
       }
     },
+    {
```

Comment:
I don't think it needs to hold this PR up, but we should make sure that there are json files that correspond to the show returns for all 3 of these, i.e, closed-with-documents-needed.json, phase3-w-documents-needed.json, phase4-w-documents-needed.json

Reply:
Noted, that'd a good point. I'll make a ticket for it.